pkg/controllers/dashboard/chart/chart.go

```diff
@@ -40,20 +51,48 @@ type Definition struct {
 	RemoveNamespace     bool
 }
 
-// RancherConfigGetter is used to get Rancher chart configuration information from the rancher config map
+// RancherConfigGetter is used to get Rancher chart configuration information from the rancher config map.
 type RancherConfigGetter struct {
 	ConfigCache corev1.ConfigMapCache
 }
 
 // GetPriorityClassName attempts to retrieve the priority class for rancher pods and feature charts as set via helm values.
 func (r *RancherConfigGetter) GetPriorityClassName() (string, error) {
```

Comment:
I'm not sure that we are getting much value out of this specific method now that we have a way to specify every value for the various sub-charts. 

I would recommend updating the original configmap to map the `priorityClass` underneath values specific to the webhook (or by specifying a "common" section applying to all charts). There's a lot of options here, but I think that the one used here could be improved on.